examples/kind/.github/conformance.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ -z "${INGRESS_CLASS}" ]; then
+  echo "Environment variable INGRESS_CLASS must be set"
+  exit 1
+fi
+
+echo "Running... (can take some time)"
+
+sonobuoy run \
+  --skip-preflight \
+  --kube-conformance-image=gcr.io/k8s-staging-ingressconformance/ingress-controller-conformance@sha256:33ad42ede4ae8b2f2f205da855bf0af12f78ec1860ca5713af5b82867235c502 \
+  --plugin-env e2e.INGRESS_CLASS=${INGRESS_CLASS}
+
+sleep 60
+
+# wait until Sonobuoy test completes
+until sonobuoy status | grep -m 1 "complete"; do : ; done
+
```

Comment:
output timestamp for each step

Reply:
Not sure it makes sense. This is the output of the command:
```
$ sonobuoy status                                                                                                            
PLUGIN		STATUS		COUNT
e2e		running		1
systemd_logs	complete	1
```

what you think should appear in the output?

Reply:
I meant -- it helps to output start and stop timestamp for long running operations so if they are part of CI, you can go back and see how long the steps tool.

Reply:
done